DigitalLearningSolutions.Web/Views/MyAccount/_EditRegistrationNumber.cshtml

```diff
@@ -4,23 +4,23 @@
 
 @{
   var errorHasOccurred = !ViewData.ModelState.IsValid;

```

Comment:
I think `errorHasOccurred` should only be true on this view if there is an error on the `HasProfessionalRegistrationNumber` property

Reply:
I have updated this to use HasProfessionalRegistrationNumber for validation